src/DacpacTool/DacpacTool.csproj

```diff
@@ -17,10 +17,10 @@
 
   <ItemGroup>
     <!-- These packages contain DacFX analysis rules, which must be located with the executable in order to be discovered-->
-    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0" IncludeAssets="runtime" />
-    <PackageReference Include="ErikEJ.DacFX.TSQLSmellSCA" Version="1.0.0" IncludeAssets="runtime" />
+    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.1" IncludeAssets="runtime" />
```

Comment:
Do we still need to include these as part of the tool?

Reply:
If we want to continue to offer them as "built-in", yes. 

Alternative is to guide users towards them in the updated docs?

Reply:
I think it makes sense to move that to the docs, and perhaps even include them by default in our templates as well, but not bloating the SDK itself with these built-in.